content/content-strategy/information-architecture/index.yml

```diff
@@ -3,6 +3,8 @@ weight: 90
 hidden: false
 theme: blue
 description: Practical guidance for government agencies to improve or develop an information architecture (IA).
+created_by: Digital Transformation Agency, Department of Agriculture and Water Resources
```

Comment:
Should be Digital Transformation Agency `and` Department of Agriculture and Water Resources